### ISSUE TITLE
OpenOCD doesn't see the STlink under WSL

### ISSUE THREAD

**Issue by Icing-weaponry** [BrianPugh/gnwmanager#93]
**Describe the bug**
When using gnwmanager under WSL-ubuntu it doesn't see the STlink probe. I get the following error message.

`raise OpenOCDAutoDetectError("Unable to connect to to debugging probe.")
gnwmanager.ocdbackend.openocd_backend.OpenOCDAutoDetectError: Unable to connect to to debugging probe.`

**To Reproduce**
Tired to flash the smw port to my zelda unit, in ubuntu WSL I ran the following command.
`make -j8 INTFLASH_BANK=2 EXTFLASH_SIZE=2621440 EXTFLASH_OFFSET=13787136 ENABLE_SAVESTATE=1 ADAPTER=stlink GNW_TARGET=zelda flash`

The same error message appears whenever I enter a gnwmanager command that tried to connect to the GnW using the STlink such as
`gnwmanager ls`

**Environment (please complete the following information):**
 - OS: Windows 10 WSL ubuntu 22.04.3 LTS
 - GnwManager Version: 0.5.2
- Debugger Probe: STlink V2 (clone)

**Additional context**
I've been migrating towards WSL to stop using my steam deck to flash the GnW.
So far I can flash a firmware, backup saves, flash ports in bank 2, everything that doesn't require gnwmanager without any issue.
The smw port now requires gnwmanager and I used it before on the steam deck and it worked fine , it is the first time I am having issues with it which leads me to belive it's wsl related

I attached a full picture of the error message below, thank you for your time.
![image](https://github.com/BrianPugh/gnwmanager/assets/65843394/ebcd932e-255e-4167-b573-1fecaddbcbe4)

**Comment by BrianPugh** [comment]
I'll be releasing a version of gnwmanager in the next day or two that has much better logging. I'll ask for a follow up then, thanks!

**Comment by BrianPugh** [comment]
sorry I forgot to followup, can you rerun this with the env variable `GNWMANAGER_VERBOSITY=debug` and post the output?

**Comment by Icing-weaponry** [comment]
Hi again,

I turned on my ubuntu WSL and powershell to connect the USB device.
I was greeted by a warning that usbipd no longer used the "wsl" flag and I had to update to usbipd-win 4.0.0
I updated and connect the USB device has normal, all goo so far.

I ran the command with GNWMANAGER_VERBOSITY=debug added and this time it worked? No issue flashing smw to bank2

So I think the issue was my version of usbipd-win being deprecated.

I apologies for the waste of time and thank you for the follow up
